### DISCUSSION TITLE
Problems in RNA and ATAC combination after SCT transformation on RNA data

### DISCUSSION THREAD

**Discussion by francescasci91** [stuart-lab/signac#1140] [Q&A] [answered]
Hi, I'm trying to combine RNA and ATAC data .I have 6 datasets of RNA and 4 datasets of ATAC. I decided to proceed in this way, also following what posted here https://github.com/satijalab/seurat/issues/5346:
1) integration of RNA datasets (using SCT transformation v2)
2) integration of ATAC datasets using the standard pipeline
3) combine the two integrated datasets obtained in point  1 and 2 to run WWN analysis.

I'm having problems with the third point.

When I try to use the `merge` function I obtain this warning 
`“Attempting to merge an SCTAssay with another Assay type Converting all to standard Assay objects.”`

And then when I try to run PCA
`Error: Cannot find 'pca' in this Seurat object`

Do you have any suggestion how to solve this??

How can I merge the two objects?

Thanks
Francesca

**Comment by timoast** [comment, marked as answer]
If you have separate RNA and ATAC datasets that you want to integrate, you can follow the steps shown here: https://satijalab.org/signac/articles/pbmc_vignette.html#create-a-gene-activity-matrix-1

WNN is not appropriate here, that method is designed for datasets where both modalities are measured in the same cell.

**Reply by francescasci91** [reply]
And if I have 4 Multiome + 2 RNA datasets, it is still advisable to use the pipeline reported here https://github.com/timoast/signac/discussions/1012 ?
So also in that case I cannot use WWN analysis but gene activity to transfer labels? 
Thanks
Francesca

**Reply by timoast** [reply]
For integrating multiome and scRNA, there's no need to use gene activities since all the datasets have RNA measured already

**Reply by nmrecka** [reply]
The link above is broken. Is there an update version of this vignette somewhere else?

**Reply by timoast** [reply]
https://stuartlab.org/signac/articles/pbmc_vignette#create-a-gene-activity-matrix
